### DISCUSSION TITLE
`@discordjs/api-extractor-*` packages aren't published

### DISCUSSION THREAD

**Discussion by pauliesnug** [discordjs/discord.js#10562] [Q&A] [answered]
### Which package is this bug report for?

discord.js

### Issue description

I wanted to include these in my projects, since I use similar TSDoc types and need some of the patches included, but it's not uploaded to npm. Could these just be published, since they need to be built within a monorepo and thus can't be included with pnpm `github:` dependencies.

### Code sample

```typescript
{
    // ./package.json
    "dependencies": {
        "@discordjs/api-extractor": "*"
    }
}
```

### Versions

- Node.js v20.18.0
- macOS Latest
- pnpm 9.12.1

### Issue priority

Low (slightly annoying)

### Which partials do you have configured?

Not applicable

### Which gateway intents are you subscribing to?

Not applicable

### I have tested this issue on a development release

62fb9de

**Comment by Qjuh** [comment, marked as answer]
What patches are we talking about here? Maybe the better route would be for us to PR those to upstream api-extractor instead.

Because this forked version is indeed custom built to work with discord.js and our specific needs, so it does contain a lot of things that won’t work/would yield wrong results for other projects.
